content/02.body.md

```diff
@@ -1,13 +1,44 @@
 ## Introduction
 
-### Node degree
-
-![Degree figure](){#fig:degree width="100%"}
-
-### Edge prediction
-
-### Feature-degree correlation
-
+Most biomedical data networks are imperfect representations of the true set of relationships.
+Real networks often mistakenly include edges that do not exist and exclude edges that do exist.
+How well a network represents the true relationships it attempts to represent depends on a number of factors, especially the methods used to generate the data in the network [@doi:10.1016/j.jprot.2014.01.020; @doi:10.3389/fgene.2015.00260; @doi:10.1038/nbt1116].
+We define "degree bias" as the type of misrepresentation that occurs when the fraction of incorrectly existent/nonexistent relationships depends on the number of connections that nodes make (their "degrees").
```

Comment:
What is the link (is there one?) between inspection bias and degree bias?

Reply:
I intended to say that inspection bias is a phenomenon that can cause degree bias. I hope my phrasing change in the sentence that follows helps to clarify this point.